doc/modules/model_evaluation.rst

```diff
@@ -1530,6 +1531,38 @@ function::
     ... # doctest: +ELLIPSIS
     0.990...
 
+.. _max_error:
+
+Max error
+-------------------
+
+The :func:`max_error` function computes the  maximum `residual error
```

Comment:
redundant space between ``the`` and ``maximum``

Reply:
Thanks. Updated.